itests/eth_deploy_test.go

```diff
@@ -0,0 +1,164 @@
+package itests
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/go-state-types/manifest"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/chain/types/ethtypes"
+	"github.com/filecoin-project/lotus/itests/kit"
+	"github.com/filecoin-project/lotus/node/config"
+)
+
+// TestDeployment smoke tests the deployment of a contract via the
+// Ethereum JSON-RPC endpoint, from an EEOA.
+func TestDeployment(t *testing.T) {
+	// TODO the contract installation and invocation can be lifted into utility methods
```

Comment:
Either implement or drop this TODO.
Also uncomment the quieting of the mining logs?

Reply:
Let's address at a later stage on master.

Reply:
Tracking as TODO::FVM